src/comments.js

```diff
@@ -933,6 +938,31 @@ function printComment(commentPath, options) {
   }
 }
 
+function isJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+  return (
+    lines.length > 1 &&
+    lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*")
+  );
+}
+
+function printJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+
+  return docBuilders.concat([
+    "/*",
+    join(
+      docBuilders.hardline,
```

Comment:
Can you remove `docBuilders.` from this line and 3 lines above?

Reply:
@lydell Done.